funnel/models/project.py

```diff
@@ -921,7 +921,9 @@ def all_unsorted(cls, legacy=None):
         """
         Return currently active events, not sorted.
         """
-        projects = cls.query.filter(cls.state.PUBLISHED)
+        from .venue import Venue
```

Comment:
This should be a tail import. Importing here will do an import every time the function is called.

Some of the other files have tail imports, for reference.